mongodb/tests/test_install.sls

```diff
@@ -0,0 +1,12 @@
+{% for package_obj in salt.pillar.get('mongodb:install:pkgs') %}
+{% for package, version in package_obj.items() %}
+test_installed_{{ package }}:
+  testinfra.package:
+    - name: {{ package }}
+    - is_installed: True
+    - version:
+        parameter: {{ version }}
+        expected: True
+        comparison: eq
+{% endfor %}
```

Comment:
If you do change the `pkgs` array to be a list of strings then you might want to leave out the version check here. Or make it a parameter with an `if` block?